_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Learning common keyboard shortcuts for screen readers"
+description: "Keyboard shortcuts are hard to learn. Here are some other ways that can help that learning."
+author: anevins12
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://dequeuniversity.com/screenreaders/
+    title: "Screen Reader Keyboard Shortcuts and Gestures"
+    source: Deque University
+  - url: https://gist.github.com/anevins12/a9cc6548fcbab5f55c1dc3c7bbb0f4bd
+    title: "Source JSON file of the mindmap that can be imported into http://app.mindmapmaker.org/"
+    source: Github
+---
+
+Have you ever felt that keyboard shortcuts for assistive technology don't stick in your head? You're not alone, I've been trying to memorise them too. Keyboard shortcuts are often presented to us in a list or a table. Lists and tables are great ways of conveying information, but sometimes a more visual approach could help us learn those shortcuts.
+
+In an ideal World we would be using common screen readers daily to help us learn through experience. When that doesn't work out, here's an approach that worked for me.
+
+## Get comfortable seeing keyboard shortcuts
+
+Most keyboard shortcuts aren't fun to look at, but we need to be comfortable reading them before taking them in. 
+
+Start by reading the lists and tables of shortcuts that people have kindly put together already. Here are a few:
+- [Deque lists common screen reader shortcuts](https://dequeuniversity.com/screenreaders/)
+- [Getting started with NVDA on The A11y Project(https://a11yproject.com/posts/getting-started-with-nvda/)]
+- [Getting started with OS X VoiceOver](Getting started with OS X VoiceOver)
+
+## Found a cool shortcut? Look it up
+At some point you've found some keyboard shortcuts that you thought were exciting. The TalkBack and VoiceOver mobile shortcuts are quite fun to try out.
+
+Start looking up videos on how to do those shortcuts. Seeing the shortcut being acted out will help that information sink in.
+
+Trying it out yourself on a device is even better. We might all have a common screen reader to hand. Android devices have TalkBack, iOS and MacOS have VoiceOver and Microsoft has Narrator.
+
+## Learning visually helps
+Sometimes there's too much information to try out and you need another way of learning the shortcuts. 
+
+Mapping the keyboard shortcuts that you've learned so far to a graph or mind map can help that information sink in. 
+
+Go back and familiarize yourself with the lists and tables of shortcuts, then start creating a visual map of them.
+
+Here's a mind map that I've created for the common keyboard shortcuts in popular screen readers: ![Mind map of common keyboard shortcuts for screen readers](/img/posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreader/common-keyboard-shortcuts-mindmap.png.png)
```

Comment:
This is a pretty complex image. We are going to need alt text beyond just saying it’s a mind map

Reply:
What do you think about using an example explanation like this after the image?

> In that mind map I've grouped shortcuts across screen readers by tasks. For example, one task is to 'list all landmarks' that branches off into JAWS, NVDA, VoiceOver iOS, TalkBack and Narrator screen readers. Each then branches off into their own shortcut:
- JAWS: R or ;
- NVDA: D
- Narrator: D
- VoiceOver iOS: Two finger twist (the rotor menu)
- TalkBack: Swipe right (the local context menu)

I was hoping that the resources linked in the previous section would be the alternative text for the map, but I don't know how to make that clear.

Reply:
I've restructured this a bit, but still the description of the image still an example. I have added a further link to Deque underneath it. Does that help or does there need to be a fuller description?